themes/baseTheme/styles/pages/PgHeaderImage.json

```diff
@@ -1,6 +1,6 @@
 {
-	"#pgHeaderImage": {
-		"image": "arrowbottom.png",
-		"imageFillType": "ASPECTFILL"
-	}
-}
\ No newline at end of file
+  "#pgHeaderImage": {
+    "image": "images://arrowbottom.png",
```

Comment:
This method might not work as expected. If this is also present on the boilerplate, it needs to be changed there too.